### ISSUE TITLE
[2759822] - Steam counter do not appear as steam counter on equipments (Only default counters)

### ISSUE THREAD

**Issue by PvtVoid** [Talishar/Talishar-FE#449]
<img width="284" alt="image" src="https://github.com/Talishar/Talishar-FE/assets/26232486/fe1a1cd0-bdca-4a38-a797-c020643dfd50">

**Comment by LaustinSpayce** [comment]
I think this one is a BE issue pls check if it's done.

**Comment by KuKetto** [comment]
Yes, this is a backend issue, it's fixed in my current branch. I will tag this when I create my pull
